src/plugin/noob_plugin.js

```diff
@@ -22,240 +23,116 @@ class NoobPluginVirtual extends EventEmitter {
   constructor (opts) {
     super()
 
-    let that = this
-    this._handle = (err) => {
-      that.emit('exception', err)
-      throw err
-    }
-
     this.id = opts.id // not used but required for compatability with five
                       // bells connector.
     this._account = opts.account
     this._prefix = null
 
     this.connected = false
 
+    // create error handling functions
+    let that = this
+    this._handle = (err) => {
```

Comment:
When you use arrow functions you don't need to keep a separate reference to `this` https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Functions/Arrow_functions